data/ka'het/ka'het missions.txt

```diff
@@ -293,3 +293,411 @@ mission "Ringworld Debris: Quarg"
+			`	"If you can bring me to the asteroid, I could be able use the controls and enter in the cupola to measure the composition of the atmosphere."`
+			choice
+				`	"I'm ready to go."`
+					accept
+				`	"Wait, are you saying that you can read the Builder language?"`
+			`	"Very approximately, but yes. We have been studying every piece of written language we could find inside their cities, mostly inside sealed containers. It is very hard to understand full phrases or even what they mean at times, but we can reliably translate most of the common words and idioms, which will hopefully be enough for our mission."`
+				accept
+	on complete
+		event "angry ka'sei"
+
+event "angry ka'sei"
+	"reputation: Ka'sei" <?= -10
+
+mission "Ka'het: Patir Mystery 5"
+	landing
+	name "First contact"
+	description "After the tumultuous first contact, return to <destination> without hurting any Ka'sei, even if they attack you."
+	source "Builder Settlement"
+	destination "Aventine"
+	passengers 1
+	to offer
+		has "Ka'het: Patir Mystery 4: done"
+	on offer
+		conversation
+			`As you approach the structure, Dusk stops preparing the equipment to observe it from the window. "It is a fascinating structure. I've only seen a small number of Builder buildings, but this looks different from anything I have seen. Perhaps it is more structurally similar to their stations than their outposts given the almost negligible gravity, or they had other asteroid colonies that have been gradually lost over time. It would be worth doing a serious exploration of the asteroid belts in the Graveyard at some point.`
+			choice
+				`	"You have been on Builder planets?"`
+					goto planets
+				`	"How far have the Remnant explored the Graveyard?"`
+				`	"Do you have have all the equipment ready?"`
+					goto dontcare
+			`	"I presume Taely already told you about the troubled beginnings we had in exploring the area; as a consequence of that, we had to reevaluate the procedures for exploring the nearby systems and species, especially beyond the Ossipago wormhole. Every scout ship entering the area has to always remain cloaked to prevent attacks from the countless Ka'het patrolling the area; this means that the further we need to travel to reach the destination, the more fuel we have to spend to return home which impacts considerably the time we can spend cloaked in the destination. Some of the systems at the other end of the Graveyard have seen very few Starlings, most of them staying for less than a day.`
```

Comment:
```suggestion
			`	"I presume Taely already told you about the troubled beginnings we had in exploring the area; as a consequence of that, we had to reevaluate the procedures for exploring the nearby systems and species, especially beyond the Ossipago wormhole. Every scout ship entering the area has to always remain cloaked to prevent attacks from the countless Ka'het patrolling the area; this means that the further we need to travel to reach the destination, the more fuel we have to spend to return home, which considerably impacts the time we can spend cloaked in the destination. Some of the systems at the other end of the Graveyard have seen very few Starlings, most of them staying for less than a day.`
```